app/controllers/login.js

```diff
@@ -0,0 +1,79 @@
+import Ember from 'ember'
+import md5 from 'npm:blueimp-md5'
+
+export default Ember.Controller.extend({
+  loginModel: {
+    properties: {
+      username: {
+        type: 'string'
+      },
+      password: {
+        type: 'string'
+      }
+    },
+    type: 'object'
+  },
+  loginView: {
+    containers: [
+      {
+        id: 'main',
+        rows: [
+          [
+            {
+              label: 'Username',
+              model: 'username'
+            }
+          ],
+          [
+            {
+              model: 'password',
+              properties: {
+                type: 'password'
+              }
+            }
+          ]
+        ]
+      }
+    ],
+    rootContainers: [
+      {
+        container: 'main',
+        label: 'Main'
+      }
+    ],
+    type: 'form',
+    version: '1.0'
+  },
+  isFormInvalid: true,
+  actions: {
+    formChange (value) {
```

Comment:
It's our standard to use JSDOCs for _all_ functions written in javascript. In this case, you should do something like:
```
/**
 * Changes the form value on the screen
 * @param {String} value - the value the user typed in
 */
```